Release-Notes/Release2.md

```diff
@@ -116,7 +116,7 @@ allowing for long-living management nodes and workload clusters managed by them.
 the same name in the manager environment (manager-configuration.yml). All configuration
 parameters from environments/configuration.yml should be moved to environments/manager/configuration.yml.
 
-* In kolla-ansible the haproxy role was renamed to loadbalancer. Accordingly, loadbalancer must now be
+* In kolla-ansible the HAProxy role was renamed to loadbalancer. Accordingly, loadbalancer must now be
```

Comment:
Sorry to be picky about this: haproxy (that is - according to this link: https://github.com/openstack/kolla-ansible/blob/master/releasenotes/notes/drop-group-haproxy-6119957627811873.yaml)